bindings/python/test/environment/atmospheric/earth/test_manager.py

```diff
@@ -0,0 +1,204 @@
+# Apache License 2.0
+
+import pytest
+
+from datetime import datetime
+import pathlib
+import os
+
+from ostk.core.filesystem import Path
+from ostk.core.filesystem import File
+from ostk.core.filesystem import Directory
+
+from ostk.io import URL
+
+from ostk.physics.time import Scale
+from ostk.physics.time import Instant
+from ostk.physics.time import Duration
+
+from ostk.physics.environment.atmospheric.earth import Manager
+from ostk.physics.environment.atmospheric.earth import CSSISpaceWeather
+
+
+@pytest.fixture
+def manager() -> Manager:
+    manager = Manager.get()
+
+    manager.set_mode(Manager.Mode.Automatic)
+    manager.set_remote_url(
+        URL.parse("https://celestrak.org/SpaceData/")
+    )  # SW-Last5Years.csv
+
+    yield manager
+
+    manager.reset()
+    manager.clear_local_repository()
```

Comment:
Move to the conftest.py?

Reply:
Fair point, as long as it's in earth it should be okay?

Reply:
Good idea